### DISCUSSION TITLE
SOC more than 100%

### DISCUSSION THREAD

**Discussion by squowse** [Louisvdw/dbus-serialbattery#1039] [General]
Does this mean anything to anyone? Immediately after this the cerbo wouldnt connect to the battery for about 6 hours.
I wonder if that coincided with the soc being below 100%

![IMG_5092](https://github.com/Louisvdw/dbus-serialbattery/assets/32992135/4bfc655b-9ba1-4da5-b9b0-9ca32d813084)

**Comment by mr-manuel** [comment]
With the latest driver the driver does not start, if the BMS reports a SOC without the real range 0-100.

**Reply by squowse** [reply]
Thanks, that makes sense. I wonder why the bms would be reporting 101%. I've never seen it do this on the bluetooth app.

**Reply by mr-manuel** [reply]
What BMS are you using?

**Reply by squowse** [reply]
Its a jbd bms.
![IMG_5108](https://github.com/Louisvdw/dbus-serialbattery/assets/32992135/f5bb5110-351a-4e04-92ce-4c9f0ce5a012)
![IMG_5107](https://github.com/Louisvdw/dbus-serialbattery/assets/32992135/075438e5-889b-422d-85dc-921943ef8dc5)

**Reply by mr-manuel** [reply]
Somewhere in the issues or discussions was reported that, if I remember correctly. Since I can't find it right now, try to sesrch for it.

**Reply by squowse** [reply]
Was it this issue? https://github.com/Louisvdw/dbus-serialbattery/issues/729

**Reply by squowse** [reply]
Should the consumed amp hours be negative as well as positive?
![IMG_5110](https://github.com/Louisvdw/dbus-serialbattery/assets/32992135/3fea7ed1-f4fe-4692-9cbc-caa8659546c5)

**Reply by mr-manuel** [reply]
No, it should only be positive

**Reply by squowse** [reply]
Ok i wonder how this is happening. The consumed ah is generated by the driver i believe? Maybe the soc over 100% is causing it? The soc comes directly from the bms or is it calcuated by the driver at all?
